components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -67,6 +67,9 @@
             while (resultSet.next()) {
                 String group = resultSet.getString(1);
                 if (StringUtils.isNotEmpty(group)) {
+                    if (StringUtils.isNotEmpty(group)) {
```

Comment:
Isn't the same condition duplicated here

Reply:
Fixed